cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,14 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {user}").format(user=user_username)}
```

Comment:
Can we just make this `username=username`?

Reply:
done